src/modules/landing/components/App/App.tsx

```diff
@@ -0,0 +1,56 @@
+import React, { useState } from 'react';
+import cn from 'classnames';
+import { Fade } from '@material-ui/core';
+
+import Header from '../Header/Header';
+
+import { useStyles } from './Styles';
+import Title from '../Title/Title';
+import WhySection from '../WhySection/WhySection';
+import Products from '../Products/Products';
+import Partners from '../Partners/Partners';
+import Footer from '../Footer/Footer';
+import FAQ from '../FAQ/FAQ';
+import BgVideo from '../BgVideo/BgVideo';
+import Menu from '../Menu/Menu';
+
+const App = () => {
+  const classes = useStyles();
+  const [isMenuOpened, setIsMenuOpened] = useState(false);
+
+  const handleMenuToggle = () => {
+    setIsMenuOpened(v => !v);
```

Comment:
```suggestion
    setIsMenuOpened(isMenuOpened => !isMenuOpened);
```

plz make ur code more readable for others

Reply:
setState is asynchronous so we should use callback in toggle - it is best practice